src/StatusQ/Components/StatusMessage.qml

```diff
@@ -37,6 +37,7 @@ Rectangle {
     property string errorLoadingImageText: ""
     property string audioMessageInfoText: ""
     property string pinnedMsgInfoText: ""
+    property string messageAttachments: ""
```

Comment:
Any reason why this is not an array? (I assume it's because NIM doesn't support array-like Qt properties? :/

Reply:
Correcto